modules/installation/pages/kubernetes-vm.adoc

```diff
@@ -0,0 +1,40 @@
+= Kubernetes VM
+:toc:
+:toclevels: 3
+
+WARNING: This page is a work in progress. *This page will be automated, instead the user will be prompted before rebooting
+to disk or right after.*
+
+. Copy the kubernetes VM image from the Live CD into the appropriate location.
++
+[source,bash]
+----
+scp /data/kubernetes-vm-x86_64.qcow2 admin@10.17.37.2:/vms/images/
+----
+. Login to the Management VM
+There are two options to get started. If you're still on the 1st Hypervisor, you can login using the management-network. The address of the first management-vm will be 10.17.37.2.
+If you are no longer on the first hypervisor, then you will need to connect using the IP address the management-vm was given at boot.
```

Comment:
Use a prescriptive tone, remove "If" and "you're/you/your".

e.g.

There are two options to get started. When launching on the first hypervisor node, the VM will be reachable via the management-network.
When running on any subsequent hypervisor node, use the IP the management-vm was given at boot.